Dockerfile.rhel

```diff
@@ -1,9 +1,9 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
```

Comment:
This dockerfile should be auto updated but the ART tooling, so no need to update this here (it seems to be out of date anyway, might need a rebase?)